### PR TITLE
[stable/owncloud] Support tls termination and existingClaim

### PR DESCRIPTION
**Pull request by yuanying**
* Support tls termination for Ingress
* Support existingClaim for apache and owncloud PVC

### CODE REVIEW

**Comment by timstoop** [comment]
I think both changes are small, obvious and no problem at all. However, it may be better in the future to make it two separate PRs.

File: stable/owncloud/README.md [hunk 1]
```diff
@@ -51,6 +51,10 @@ The following tables lists the configurable parameters of the ownCloud chart and
 | `imagePullPolicy`                   | Image pull policy                         | `IfNotPresent`                              |
 | `ingress.enabled`                   | Enable use of ingress controllers         | `false`                                     |
 | `ingress.servicePort`               | Ingress' backend servicePort              | `http`                                      |
+| `ingress.annotations`               | An array of service annotations           | `nil`                                       |
+| `ingress.tls[i].secretName          | The secret kubernetes.io/tls              | `nil`                                       |
```
Comment (unguiculus) [inline stable/owncloud/README.md]: I think this is confusing. Just add one line for `ingress.tls`.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR.

File: stable/owncloud/README.md (hunk 1 above)
Comment (yuanying) [reply stable/owncloud/README.md]: Fixed

**Comment by yuanying** [comment]
Sorry for late reply.

@timstoop Thanks your suggestion! I'll do at my later PR.
@unguiculus Done!

**Review by unguiculus — APPROVED** [review]

**Pull request merged**